### PR TITLE
Adding hasasia a pulsar timing array sensitivity Python package

### PR DESCRIPTION
**Pull request by Hazboun6**
A package for calculating sensitivity of pulsar timing array experiments to gravitational waves from supermassive binary black holes.

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/hasasia/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 87ab202bb6c07b24922169fc12115f9fe074c9d6ae33df08b91cb3c4ca48436f
+
+build:
+  noarch: python
+  number: 0
+  skip: True  # [py<35]
```
Comment (dopplershift) [inline recipes/hasasia/meta.yaml:15]: This looks like pure Python so could be noarch Python (and reduce the number of builds):
```suggestion
  noarch: python
```

File: recipes/hasasia/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
```
Comment (dopplershift) [inline recipes/hasasia/meta.yaml:20]: For noarch:
```suggestion
    - python >=3.5
```

File: recipes/hasasia/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    - pip
+  run:
+    - python
```
Comment (dopplershift) [inline recipes/hasasia/meta.yaml:23]: For noarch:
```suggestion
    - python >=3.5
```

**Review by dopplershift — COMMENTED** [review]
Looks pretty good, but I think this can be noarch Python?

File: recipes/hasasia/meta.yaml (hunk 3 above)
Comment (Hazboun6) [reply recipes/hasasia/meta.yaml]: @dopplershift Thanks for the suggestions!

File: recipes/hasasia/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    - pip
+  run:
+    - python >=3.5
+    - numpy >=1.14
+    - scipy >=1.0
+    - astropy >=3.0
+
+test:
+  imports:
+    - hasasia
```
Comment (dopplershift) [inline recipes/hasasia/meta.yaml:29]: I just noticed that `hasasia.__init__` basically doesn't do anything. Can you add some more imports here that verify the package isn't broken. Maybe like?
```suggestion
    - hasasia
    - hasasia.sensitivity
```

**Review by dopplershift — CHANGES_REQUESTED** [review]

File: recipes/hasasia/meta.yaml (hunk 4 above)
Comment (Hazboun6) [reply recipes/hasasia/meta.yaml:29]: Okay, I've now imported all of the submodules.

**Review by dopplershift — APPROVED** [review]

**Pull request merged**